app/resources/admin.html.template

```diff
@@ -39,17 +39,18 @@
 <h2>Select a repository</h2>
 
 <script>
-  var LANGUAGE_EXONYMS = {{language_exonyms_json|safe}};
 
-  /* Returns true if the array contains elem */
-  function arr_contains(arr, elem) {
-    for (var i = 0; i < this.length; i++) {
-      if (arr[i] === elem) {
-        return true;
-      }
-    }
-    return false;
-  }
+//TODO: move to object-oriented!!
+
+  var LANGUAGE_EXONYMS = {{language_exonyms_json|safe}};
+  var CUSTOM_MESSAGE_TABLE_IDS = [
+    'repo_titles', 
```

Comment:
We probably should remove 'repo_titles' from CUSTOM_MESSAGE_TABLE_IDS? Then we can keep the const name as is. Since you have a lot of special casing for 'repo_titles', it doesn't probably make sense to be a member of this list.